internal/protocol/request.go

```diff
@@ -7,27 +7,31 @@ package protocol
 
 // EncodeLeader encodes a Leader request.
 func EncodeLeader(request *Message) {
+    request.Reset()
```

Comment:
Is it just github looking confused or is there really something weird going on here with tab alignment?

Reply:
Ah it's because this code is automatically generated with ```go generate``` (see ```schema.sh```), and I used spaces instead of tab for indentation in the generation script.

Fixed now.